supervisord.conf

```diff
@@ -9,8 +9,7 @@ supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 
 [program:jenkins]
 command=/usr/local/bin/shim-startup-wrapper.sh
-directory=/var/jenkins_home
-environment=JENKINS_HOME=/var/jenkins_home
```

Comment:
Removed this ^ since supervisord is documented and does inherit those, so it's a duplicate of the JENKINS_HOME value we better avoid.